spec/controllers/posts_controller_spec.rb

```diff
@@ -19,10 +24,12 @@
       post = create(:post)
       expect(Post.find_by(message: "Hello, world!")).to be
```

Comment:
it was a default test, you just expect the object to exist. I don't think we need it anymore though